internal/data/payments_test.go

```diff
@@ -1205,3 +1205,159 @@ func Test_PaymentModelCancelPayment(t *testing.T) {
 		assert.Equal(t, CanceledPaymentStatus, payment2DB.Status)
 	})
 }
+
+func Test_PaymentModel_GetReadyByDisbursementID(t *testing.T) {
+	dbt := dbtest.Open(t)
+	defer dbt.Close()
+
+	dbConnectionPool, err := db.OpenDBConnectionPool(dbt.DSN)
+	require.NoError(t, err)
+	defer dbConnectionPool.Close()
+
+	ctx := context.Background()
+
+	models, err := NewModels(dbConnectionPool)
+	require.NoError(t, err)
+
+	DeleteAllPaymentsFixtures(t, ctx, dbConnectionPool)
+	DeleteAllDisbursementFixtures(t, ctx, dbConnectionPool)
+	DeleteAllCountryFixtures(t, ctx, dbConnectionPool)
+	DeleteAllAssetFixtures(t, ctx, dbConnectionPool)
+	DeleteAllReceiverWalletsFixtures(t, ctx, dbConnectionPool)
+	DeleteAllReceiversFixtures(t, ctx, dbConnectionPool)
+	DeleteAllWalletFixtures(t, ctx, dbConnectionPool)
```

Comment:
Why do you need this? Tests run in a brand new database, so I don't think there's anything there that needs to be deleted, right?